pillar/devstack.sls

```diff
@@ -274,8 +252,8 @@ edx:
     EDXAPP_LMS_ENV_EXTRA:
       <<: *common_env_config
       BULK_EMAIL_DEFAULT_FROM_EMAIL: {{ DEFAULT_FEEDBACK_EMAIL }}
-      COURSE_ABOUT_VISIBILITY_PERMISSION: "{{ edx.edxapp_course_about_visibility_permission }}"
-      COURSE_CATALOG_VISIBILITY_PERMISSION: "{{ edx.edxapp_course_catalog_visibility_permission }}"
+      COURSE_ABOUT_VISIBILITY_PERMISSION: "staff"
+      COURSE_CATALOG_VISIBILITY_PERMISSION: "staff"
```

Comment:
We should just remove this line. That setting is for limiting access for students on the staging servers, not necessary in a devstack environment.